# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1544727879 18000
#      Thu Dec 13 14:04:39 2018 -0500
# Node ID 07e60035b09bcbe1d14e9ed436125966394ed66a
# Parent  db43bd1fc038130a6e22ac619706f2ff58012c1f
maint: Update gnulib to latest version.

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-e08cdbed0c7479ef75a3c08807bfbb1f64242db5 gnulib
+a4bc639f000375e3522c56509d30d51a2740694b gnulib
